bones/exemplar.go

```diff
@@ -0,0 +1,235 @@
+package bones
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+// MethodExemplar returns an exemplar for a method, with an exemplar for the
+// input message as a comment and a function returning an exemplar of the
+// output message as the method implementation.
+func MethodExemplar(md protoreflect.MethodDescriptor) exemplar {
+	// Format the input message exemplar as a comment
+	ime := MessageExemplar(md.Input())
+	if md.IsStreamingClient() {
```

Comment:
This needs to also check `!md.IsStreamingServer()` since for bidi streams, we still use the `request:` field as we call the method once for each message on the client stream.

Reply:
Fixed